scripts/s3mi

```diff
@@ -326,14 +338,46 @@ def adjust_RAM_params():
         MAX_CONCURRENT_REQUESTS = 15
 
 
-def main_cat(s3_uri):
+def get_credentials():
```

Comment:
Question for @kislyuk and @davidrissato --- if this fails, should it be retried?

I am being careful not to worse s3mi's reliability.